xclim/core/checks.py

```diff
@@ -383,3 +390,27 @@ def missing_pct(da, freq, tolerance, **indexer):
       A boolean array set to True if period has missing values.
     """
     return MissingPct(da, freq, **indexer)(tolerance=tolerance)
+
+
+def at_least_n_valid(da, freq, n, **indexer):
```

Comment:
I think it would be better to have a matching name like `missing_????`, but I can't think of one so ok.

Reply:
@tlogan2000 Ideas ?